packages/api-plugin-sequences/src/registration.js

```diff
@@ -0,0 +1,12 @@
+export const Sequences = [];
```

Comment:
Optional naming suggestion. `Sequences` looks like a collection name to me.

```suggestion
export const sequenceConfigs = [];
```

Reply:
Changed